src/index.js

```diff
@@ -1,3 +1,4 @@
+const os = require('os');
```

Comment:
@dnalborczyk will this work in the browser environment? Does webpack polyfill this when prepublish is run?